grammars/go.cson

```diff
@@ -128,6 +128,10 @@
     'match': '\\b(iota|true|false|nil)\\b'
   }
   {
+    'match': '\\;'
+    'name': 'punctuation.terminator.statement.go'
```

Comment:
I would say we can lose the delimiter part as it's clearly not a delimiter: `meta.terminator.semicolon.go`

Next to that it doesn't much sense to make these `meta` scope names longer as needed right...